tests/main/nfs-support/task.yaml

```diff
@@ -185,7 +185,7 @@ execute: |
     # not supported. Error displayed:
     # - arch: mount.nfs: requested NFS version or transport protocol is not supported
     # - debian-sid, ubuntu-20.10: mount.nfs: an incorrect mount option was specified
-    if [[ "$SPREAD_SYSTEM" != arch-* && "$SPREAD_SYSTEM" != debian-sid-* && "$SPREAD_SYSTEM" != ubuntu-20.10-* ]]; then
+    if not os.query is-arch-linux && "$SPREAD_SYSTEM" != debian-sid-* && [[ "$SPREAD_SYSTEM" != ubuntu-20.10-* ]]; then
```

Comment:
Why is it mixing the new and old style checks?

Reply:
we will have new commands for os.query on next PR, the idea will have commands for other system versions like debian-sid, in the meantime both ways will coexist

Reply:
It is just to avoid making a very big change